Locales/frFR.lua

```diff
@@ -4,31 +4,31 @@ local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Nouvelles invasions disponibles !"
+L.tooltipClick = "|cffeda55fClick|r pour glisser et déplacer."
+L.tooltipClickOptions = "|cffeda55fRight-Click|r pour ouvrir les options."
```

Comment:
This is missing the translation for "right-click". 

What about the remaining English words, did you struggle to translate them?